src/compiler/asn_compiler.yacc

```diff
@@ -1914,7 +1914,7 @@ re2c:define:YYCURSOR = "context.cursor";
                         { context.location.columns(context.cursor - start); return yylex(context); }
 "--" ([\-]?[^\r\n\-])*[\-]?
                         { context.location.columns(context.cursor - start); return yylex(context); }
-"/*" ([^\*]|[\*][^/])* "*/"
+"/*" ([^\*]|([\*]+[^\*\/]))*[\*]+ "/"
```

Comment:
I'm sure it's a good fix, do you have an example of the problem we could add to testfiles?

https://github.com/Samuel-Tyler/fast_ber/tree/master/testfiles

Reply:
Another small point is that  the file `src/compiler/autogen_copy/asn_compiler.hpp` must be updated when .yacc is updated. This is a copy used for windows builds where bison does not run.

Reply:
> I'm sure it's a good fix, do you have an example of the problem we could add to testfiles?
> 
> https://github.com/Samuel-Tyler/fast_ber/tree/master/testfiles
> 
> (Just to prevent it happening in future)

A comment written like this made the compiler segfault:
```
/************
-- OIDs
************/
```
Just by adding a space before the closing tag resolved the segfault issue:

```
/************
-- OIDs
*********** */
```